mozdownload/scraper.py

```diff
@@ -364,6 +365,14 @@ def _download():
 
         return self.filename
 
+    def get_file_extension(self, binary):
+        suffixes = Path(binary).suffixes
+        if suffixes[-1] in [".bz2", ".xz"]:
```

Comment:
Actually we also need to support `.tar.gz` which I missed. Also we should change the logic here given that `.tar` isn't required to be always present.

Reply:
This was kinda crappy. With the updated code we do not need this anymore.